### ISSUE TITLE
Font flickers when changing page

### ISSUE THREAD

**Issue by miquelromero** [nuxt-community/google-fonts-module#20]
This bug has been introduced in version 1.1.0, here's a codesandbox link:
https://codesandbox.io/s/confident-rgb-sj4lm?file=/pages/about.vue

When navigating between pages, the font gets unavailable for some milliseconds.

Thanks!

EDIT: Open chrome inspector to make it more noticeable

**Comment by Atinux** [comment]
Confirmed @ricardogobbosouza, we have the issue currently on content.nuxtjs.org

**Comment by ivodolenc** [comment]
Hi, this may not be directly related, but I tried this code below in a new empty project (npx create nuxt app) and it works as expected in dev mode without font flickering, but in production mode (npm run generate) it doesn't work. In production, font flickering is still present when changing the route.

Also in dev & prod mode, after each route change, the tags with `once: true` are doubled and resource consumption is increased. [#620](https://github.com/nuxt/vue-meta/issues/620)

```js
// nuxt.config.js
{
  head: {
    link: [
      {
        once: true, // Following @ricardogobbosouza's example
        rel: 'preload',
        href: '/fonts/font-face.css',
        as: 'style',
      },
      {
        once: true, // Following @ricardogobbosouza's example
        rel: 'stylesheet',
        href: '/fonts/font-face.css',
        media: 'print',
        onload: "this.onload=null;this.removeAttribute('media')",
      },
    ]
  }
}
```

- `npx create nuxt app`
- `nuxt:` '2.14.6' & '2.14.7'
- `target:` 'static'

**Comment by ricardogobbosouza** [comment]
Thanks @ivodolenc 
I will analyze

**Comment by miquelromero** [comment]
@Atinux flickering is still happening in 1.1.2
https://codesandbox.io/s/confident-rgb-sj4lm

**Comment by Atinux** [comment]
Cc @ricardogobbosouza

**Comment by ricardogobbosouza** [comment]
Fixed on [v1.1.3](https://github.com/nuxt-community/google-fonts-module/releases/tag/v1.1.3)

**Comment by lostful** [comment]
This is still an issue
